packages/cdktf/lib/terraform-stack.ts

```diff
@@ -45,6 +45,14 @@ function throwIfIdIsGlobCharacter(str: string): void {
   });
 }
 
+function throwIfIdContainsWhitespace(str: string): void {
+  if (str.includes(" ")) {
```

Comment:
Should we extend this to any whitespace character while were at it?
Maybe use a regex with `\s`?